cmake/z3.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ba3ec7bce..7ca1894fd 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -446,13 +446,13 @@ configure_file(
+ 
+ # Target needs to be declared before the components so that they can add
+ # dependencies to this target so they can run their own custom uninstall rules.
+-add_custom_target(uninstall
+-  COMMAND
+-  "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
+-  COMMENT "Uninstalling..."
+-  USES_TERMINAL
+-  VERBATIM
+-)
++# add_custom_target(uninstall
++#   COMMAND
++#   "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
++#   COMMENT "Uninstalling..."
++#   USES_TERMINAL
++#   VERBATIM
++# )
+ 
+ ################################################################################
+ # CMake build file locations
+diff --git a/src/util/memory_manager.cpp b/src/util/memory_manager.cpp
+index 290881ba5..a3db94d7a 100644
+--- a/src/util/memory_manager.cpp
++++ b/src/util/memory_manager.cpp
+@@ -29,6 +29,8 @@ Copyright (c) 2015 Microsoft Corporation
+ # define malloc_usable_size _msize
+ #endif
+ 
++#undef HAS_MALLOC_USABLE_SIZE
```

Comment:
It would be good to add a comment here why it is is needed. I'd just add normal C++ comment into the patch file.